examples/jvm/defaults/build.gradle.kts

```diff
@@ -0,0 +1,86 @@
+plugins {
+    kotlin("jvm") version "1.7.10"
+    id("org.jetbrains.kotlinx.kover") version "0.6.1"
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation(kotlin("test"))
+}
+
+kover {
+    isDisabled = false
+
+    useKoverToolDefault()
+
+    excludeInstrumentation {
+        className("com.example.subpackage.*")
+    }
+}
+
+koverReport {
+    filters {
+        excludes {
+            className("com.example.subpackage.*")
+        }
+        includes {
+            className("com.example.*")
+        }
+    }
+
+    xml {
+        onCheck = false
+        setReportFile(layout.buildDirectory.file("my-project-report/result.xml"))
+
+        filters {
+            excludes {
+                className("com.example2.subpackage.*")
+            }
+            includes {
+                className("com.example2.*")
```

Comment:
Also, I wonder if `name` is redundant.

Maybe omitting it will be even more concise?
```
includes {
    classes("foo.*",  "bar.*")
    packages("com.foo")
}
```

WDYT?

Reply:
If call write
```
includes {
    classes("foo.*",  "bar.*")
    packages("com.foo")
}
```

How do add a filter to annotations?
Is it ok?
```
includes {
    classes("foo.*",  "bar.*")
    packages("com.foo")
    annotatedBy("*Generated*")
}
```